### PR TITLE
Feature: Payment

### PR DESCRIPTION
**Pull request by vietanhdev**

### CODE REVIEW

File: src/components/cart/Payment.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+export default function Payment({  lng, handleCancel, handlePay, orderId,  orderAmount}) {
+    const [showError, setShowError] = useState(false);
+    const { t } = useTranslation(lng, "common");
+    return (
+        <div className="flex-col">
+        <span className="text-xl mb-4">
+            Scan to pay
+        </span>
+        <img src={`https://api.vietqr.io/image/970415-100879113533-uKJAdJQ.jpg?accountName=TRINH%20THU%20HAI&amount=1000&addInfo=PAY%20ORDER%20CAFETERIA${orderId}`} className="w-60 h-auto mb-4" />
```
Comment (vietanhdev) [inline src/components/cart/Payment.js:13]: @haitt00 Use process.env for variables.

File: src/app/[lng]/cart/page.js [hunk 2]
```diff
@@ -68,8 +93,11 @@ export default function Cart({ params: { lng } }) {
       deliveryAddress,
       storeId,
       (data) => {
-        localStorage.setItem("cart", "[]");
-        router.push(`/${lng}/orders`);
+        // localStorage.setItem("cart", "[]");
```
Comment (vietanhdev) [inline src/app/[lng]/cart/page.js:96]: Clean up if not used.

File: src/pages/api/payment/[...param].js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          return;
+        }
+        const amount = docSnap.data().totalPrice
+        // if(amount == req.body.amount) {
+            await updateDoc(docRef, {
+                paid: true,
+            });
+        // }
+
+        //add transaction info to db for refund
```
Comment (vietanhdev) [inline src/pages/api/payment/[...param].js:25]: Clean up if not used.

**Review by BangMach — APPROVED** [review]
Accept new changes.

**Pull request merged**